### PR TITLE
refactor(tree-wide): add missing `:root` to 9 styles

### PR DESCRIPTION
**Pull request by thismoon**
## 🔧 What does this fix? 🔧

userstyles that depend on the user's system theme use this snippet from the template style:
```less
  @media (prefers-color-scheme: light) {
    :root {
      #catppuccin(@lightFlavor, @accentColor);
    }
  }
  @media (prefers-color-scheme: dark) {
    :root {
      #catppuccin(@darkFlavor, @accentColor);
    }
  }
```
however some userstyles doesn't have the `:root` part which results in the selection background color not working

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/pinterest/catppuccin.user.css
Comment (isabelroses) [inline styles/pinterest/catppuccin.user.css:1]: Can you revert this file, changes here seem to be breaking the style.

**Review by isabelroses — CHANGES_REQUESTED** [review]

File: styles/pinterest/catppuccin.user.css
Comment (thismoon) [reply styles/pinterest/catppuccin.user.css:1]: i found a way to fix the selection without breaking the style (https://github.com/catppuccin/userstyles/pull/1302/commits/ac8061bafa1da676256dac794d21479c5ebdd3b0)
is it okay to keep the fix in this pr?

**Review by isabelroses — DISMISSED** [review]

File: styles/pinterest/catppuccin.user.css [hunk 1]
```diff
@@ -164,6 +160,9 @@
     /* the void (parts not loaded yet) */
     html {
```
Comment (uncenter) [inline styles/pinterest/catppuccin.user.css:161]: Instead, change the selectors above to use `:root` like you have been, and replace the `html` here (and any other `html`s in this file) with `&`.

**Review by Guaxinim5573 — DISMISSED** [review]

**Review by uncenter — APPROVED** [review]
Assuming you have tested each of these, looks good!

**Review by AnubisNekhet — APPROVED** [review]

**Pull request merged**